be/src/exec/tablet_sink.cpp

```diff
@@ -44,6 +46,8 @@
 #include "util/monotime.h"
 #include "util/uid_util.h"
 
+static bvar::Adder<int> g_tablet_sink_pending_chunk("starrocks", "tablet_sink_pending_chunks");
```

Comment:
I think we'd better to define metrics in one place.
And more comments for each metric is need.